templates/shared/pack.js

```diff
@@ -0,0 +1,12 @@
+const { readFileSync } = require('fs');
+const { parse, resolve } = require('path');
+const AdmZip = require('adm-zip');
+
+const { base } = parse(__dirname);
+const { version } = JSON.parse(
+  readFileSync(resolve(__dirname, 'build', 'manifest.json'), 'utf8')
+);
+
+const zip = new AdmZip();
+zip.addLocalFolder('build');
+zip.writeZip(`${base}-v${version}.zip`);
```

Comment:
Can we generate this zip file inside a `release` directory?
And ignore the `release` directory in `.gitignore` file so that it won't get accidentally committed.